go.mod

```diff
@@ -16,9 +16,9 @@ require (
 	github.com/spf13/afero v1.3.4
 	github.com/ulikunitz/xz v0.5.7 // indirect
 	gopkg.in/yaml.v2 v2.3.0
-	helm.sh/helm/v3 v3.1.3
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
+	helm.sh/helm/v3 v3.2.4
+	k8s.io/apimachinery v0.18.5
+	k8s.io/client-go v0.18.5
 	rsc.io/letsencrypt v0.0.3 // indirect
-	sigs.k8s.io/controller-runtime v0.4.0
+	sigs.k8s.io/controller-runtime v0.6.1
```

Comment:
Are we sure we can go that far? I remember there were breaking changes and that may just causes more disruption in the operators. If the new k8s deps sort this then this is super cool. Just wanted to raise a thought here.

Reply:
This was outdated. In operatorkit we're using v0.5.2. But we need to use v0.6.X for K8s 1.18 support.

We used v0.6.1 in all the test projects and it has been OK.